src/components/FilterList.jsx

```diff
@@ -0,0 +1,20 @@
+import PropTypes from "prop-types";
```

Comment:
@tmccoy529 I dont think this is being used?

Reply:
Yeah youre right. That component is dead. Its been removed